api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -610,10 +611,12 @@ public void syncTwoFactorAuthStatus() {
 
   @Override
   public DbUser syncTwoFactorAuthStatus(DbUser targetUser, Agent agent) {
-    return syncTwoFactorAuthStatus(
-        targetUser,
-        agent,
-        directoryService.getUserOrThrow(targetUser.getUsername()).getIsEnrolledIn2Sv());
+    boolean isEnrolledIn2FA =
+        directoryService
+            .getUser(targetUser.getUsername())
```

Comment:
while it's unclear how a user can be "not found" by Google Directory service, this means that we can't confirm their 2FA status.  For compliance, we should proactively restrict their access instead of throwing here.